core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -212,7 +226,7 @@ class BpmBillingProjectLifecycle(
   private def cleanupBillingProfile(profileModelId: UUID,
                                     projectName: RawlsBillingProjectName,
                                     ctx: RawlsRequestContext
-  ): Future[Unit] = {
+  ): Future[Boolean] = {
```

Comment:
I think I changed this to return a boolean just to make some debugging quicker - it seems harmless to leave it in

Reply:
But I find it confusing to understand why it is returning true vs. false. If the boolean stays, you need to update the documentation to explain what it means.

Reply:
@blakery I still thinks this should be backed out, or the meaning of the boolean should be documented.

Reply:
I didn't see your comment earlier - removed the return value

Reply:
I recall there was a also a test that returned a value because of this.

Reply:
There shouldn't be; it's a private method.

Reply:
Ugh, link isn't working… I think because the file is collapsed by default. I'll find it again and put a comment directly there.